packages/upstream-protobuf/index.mjs

```diff
@@ -199,7 +195,6 @@ export class UpstreamProtobuf {
     try {
       writeTree(Object.entries(files), tempDir);
       const args = [
-        "--experimental_editions",
```

Comment:
Editions are no longer experimental, and the experimental prefix is no longer supported.